src/gui/src/AboutDialog.cpp

```diff
@@ -49,6 +49,8 @@ AboutDialog::AboutDialog(QWidget* parent)
 	about += tr( "<b>Project page</b><br>");
 	about += "http://sourceforge.net/projects/hydrogen<br><br>";
 	about += tr( "<b>Mailing lists</b><br>");
+	about += "http://lists.sourceforge.net/lists/listinfo/hydrogen-users<br>";
+	about += "http://lists.sourceforge.net/lists/listinfo/hydrogen-announce<br>";
```

Comment:
Still listed on the homepage **but** in the "developer zone" which is not the place the users are going to in the first place I guess.

That said, my first intention was to correct the fact that the code here above states "Mailing lists" (note the plural form). So I checked the ML available and I found the 2 others. Then I put these in the code in this commit.

It does look that keeping those ML is not the way you guys want to take, and I've got no opinion about that. I would only suggest here that it should be consistent across the software, the documentation and the website.

So, it does look to me that we have 2 possibilities:

1. these 2 ML are to be considered as "dead", then we should:
- remove these from the "developer zone -> ML"
- remove the plural form in this commit

2. these 2 ML are not that dead, then we should:
- move these from the "developer zone -> ML" to somewhere else where the common users will feel that this information is for them

If we agree on 1), I'm then removing these from the src/gui/src/AboutDialog.cpp and will enjoy to remove the plural form in the same time, making it clearer.

Agree ?